networkx/classes/tests/timingclasses.py

```diff
@@ -519,7 +519,7 @@ def remove_nodes_from(self, nodes):
                 pass
 
 
-    def nodes_iter(self, data=False):
+    def nodes(self, data=False):
```

Comment:
These changes were made by @jfinkels and I have reverted back the changes in #1587